.github/workflows/test.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
           fetch-depth: 0
       - uses: actions/setup-go@v3
         with:
-          go-version: 1.19
+          go-version: '1.20'
```

Comment:
might need to update https://github.com/acorn-io/acorn-dns/blob/main/.github/renovate.json for this too